.github/workflows/latest_dependency_checker.yml

```diff
@@ -8,6 +8,7 @@ on:
   workflow_dispatch:
 jobs:
   build:
+    if: ${{ github.repository_owner == 'alteryx' }}
```

Comment:
Let's do this for the 3rd scheduled workflow:
https://github.com/alteryx/featuretools/blob/main/.github/workflows/test_with_woodwork_main_branch.yml

Reply:
Updated [3e6e807](https://github.com/alteryx/featuretools/pull/1973/commits/3e6e80772cd124cb4d15043a4316a1881203b033)